intro-HPC/ch_jobscript_examples.tex

```diff
@@ -7,25 +7,36 @@ \section{Single-core job}
 
 \examplecode{bash}{single_core.sh}
 
+\begin{enumerate}
+    \item We set \lstinline|#PBS| headers, see \autoref{ch:torque-options} for a list of these options
```

Comment:
Using `#PBS` header lines, we specify the resource requirements for the job, see ...